god-mode.el

```diff
@@ -463,6 +464,70 @@ be ignored by `god-execute-with-current-bindings'."
                        universal-argument
                        universal-argument-more)))
 
+
+(defvar god-latest-described-command nil
+  "The latest command recorded by `god-mode-describe-key'.")
+
+(defun god-mode--help-fn-describe-function (_arg)
+  "Insert information about `god-mode' key-bindings for the described function.
+The argument _ARG is ignored: it's only needed because all hooks in
+`help-fns-describe-function-functions' take the function-name as argument.
+But in our case it's redundant"
+  (insert
+   (format-message "\n  %s\n  %s%s\n  %s%s%s\n\n"
+		   "(`god-local-mode' is enabled. "
+		   " The given key-sequence: "
+		   (god-mode-get-described-key-seq)
+		   " corresponds to this key-binding: "
+		   (help--key-description-fontified
+		    god-latest-described-command)
+		   ")")))
+
+(defun god-mode-get-described-key-seq ()
+  "Return the keys that were pressed after `god-mode-describe-key' was called."
+  (let* ((latest-keys (recent-keys 'include-cmds))
+	 (latest-describe-key-index
+	  (cl-position '(nil . god-mode-self-insert)
+		       latest-keys :from-end t :test #'equal))
+	 (start-index (+ 3 latest-describe-key-index))
+	 (key-sequence (key-description (seq-subseq latest-keys start-index))))
+    (propertize key-sequence
+		'font-lock-face 'help-key-binding
+                'face 'help-key-binding)))
+
+(defun god-mode-describe-key ()
+  "Describe a key-sequence as interpreted by `god-mode'.
+Prompt for a key sequence, use `god-mode-lookup-key-sequence' to translate it
+into the appropriate command, and use `describe-function' to describe it.
+Only applied when `god-translate-key-for-description' is t:
+when nil, `describe-key' is called instead"
```

Comment:
This part of the docstring that refers to `god-translate-key-for-description` is redundant now.